change/@ni-jasmine-parameterized-65d6b534-8705-4960-a2f3-3a8b73e8ba96.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Since we're only updating `devDependencies` my understanding is that this only impacts the version we build/test against but doesn't impact the published package or requirements for clients. If that's the case then I think we can set `"type": "none"` in all of these files. (It'd be good to get Milan's confirmation on this)

Reply:
That's a good point. I'll make the change, and if Milan or anyone else argues otherwise, I'll change it back.

Reply:
It has the potential to change the build output. The compiler is generating the output code (we don't ship .ts, we ship output from the tsc compiler). So a change in compiler version does impact the output an app consumes. It should be patch.